src/test/java/uk/gov/justice/digital/delius/transformers/NsiPatchRequestTransformerTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.justice.digital.delius.transformers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.digital.delius.config.DeliusIntegrationContextConfig.IntegrationContext;
+import uk.gov.justice.digital.delius.data.api.ContextlessReferralEndRequest;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class NsiPatchRequestTransformerTest {
+
+    private static final OffsetDateTime END_DATE = OffsetDateTime.of(2021,6,1,0,0,0,1, ZoneOffset.UTC);
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private IntegrationContext integrationContext;
+
+    private NsiPatchRequestTransformer transformer = new NsiPatchRequestTransformer();
+
+    @BeforeEach
+    public void before() {
+
+        integrationContext = new IntegrationContext();
+        integrationContext.getContactMapping().setEndTypeToOutcomeType(
+            new HashMap<>() {{
+                this.put("CANCELLED", "CRS01");
+                this.put("PREMATURELY_ENDED", "CRS02");
+                this.put("COMPLETED", "CRS03");
+            }}
+        );
+    }
+
+    @Test
+    public void transformsToJsonPatchMappingCorrectOutcome() throws JsonProcessingException {
+
+        assertThat(
+            objectMapper.writeValueAsString(
+                transformer.mapEndTypeToOutcomeOf(buildRequest("CANCELLED"), integrationContext)
+            )
+        ).isEqualTo("[{\"op\":\"replace\",\"path\":\"/outcome\",\"value\":\"CRS01\"}," +
```

Comment:
you could use a multiline string now since we are on Java 16 :-)

Reply:
Point taken - thx